tests/test_core.py

```diff
@@ -99,6 +99,9 @@ def metafunc(self, native_exceptions):
         self.skipTest('wasm2js does not support wasm exceptions')
       if not config.V8_ENGINE or config.V8_ENGINE not in config.JS_ENGINES:
         self.skipTest('d8 required to run wasm eh tests')
+      # FIXME Temporarily disabled. Enable this later when the bug is fixed.
+      if '-fsanitize=address' in self.emcc_args:
```

Comment:
Maybe `if any(a.startswith('-fsantitize') for a in self.emcc_args):` so that we catch lsan too?

Reply:
I ran all EH tests with LSan and the only test failing is `test_dylink_raii_exceptions_wasm_eh`, with the message 
```
em++: error: LSan does not support dynamic linking
```

All dylink tests seem to have `@needs_dylink` decorator, and [`needs_dylink`](https://github.com/emscripten-core/emscripten/blob/f2c2432c43e2392473582e31484232bea047b324/tests/runner.py#L141-L149) calls [`check_dylink`](https://github.com/emscripten-core/emscripten/blob/f2c2432c43e2392473582e31484232bea047b324/tests/runner.py#L393-L399), which only [skips](https://github.com/emscripten-core/emscripten/blob/f2c2432c43e2392473582e31484232bea047b324/tests/runner.py#L398-L399) tests for ASan.

I submitted #14038 to disable LSan for dylink tests too. Other wasm EH tests work with LSan.